design/FY2023/NFP-Space Sizing and HVAC.md

```diff
@@ -0,0 +1,514 @@
+
+Output:Variable,*,Space Predicted Moisture Load Moisture Transfer Rate,hourly; !- HVAC Average [kgWater/s]
+Output:Variable,*,Space Predicted Moisture Load to Humidifying Setpoint Moisture Transfer Rate,hourly; !- HVAC Average [kgWater/s]
+Output:Variable,*,Space Predicted Moisture Load to Dehumidifying Setpoint Moisture Transfer Rate,hourly; !- HVAC Average [kgWater/s]
+Output:Variable,*,Space System Predicted Moisture Load Moisture Transfer Rate,hourly; !- HVAC Average [kgWater/s]
+Output:Variable,*,Space System Predicted Moisture Load to Humidifying Setpoint Moisture Transfer Rate,hourly; !- HVAC Average [kgWater/s]
+Output:Variable,*,Space System Predicted Moisture Load to Dehumidifying Setpoint Moisture Transfer Rate,hourly; !- HVAC Average [kgWater/s]
+```
+
+*Space equivalents may be added for the following zone output variables (not sure about some of these):*
+
+```
+Zone,Average,Zone Mean Radiant Temperature [C]
+
+HVAC,Sum,Zone Infiltration Sensible Heat Loss Energy [J]
+HVAC,Sum,Zone Infiltration Sensible Heat Gain Energy [J]
+HVAC,Sum,Zone Infiltration Latent Heat Loss Energy [J]
+HVAC,Sum,Zone Infiltration Latent Heat Gain Energy [J]
+HVAC,Sum,Zone Infiltration Total Heat Loss Energy [J]
+HVAC,Sum,Zone Infiltration Total Heat Gain Energy [J]
+HVAC,Average,Zone Infiltration Current Density Volume Flow Rate [m3/s]
+HVAC,Average,Zone Infiltration Standard Density Volume Flow Rate [m3/s]
+HVAC,Sum,Zone Infiltration Current Density Volume [m3]
+HVAC,Sum,Zone Infiltration Standard Density Volume [m3]
+HVAC,Sum,Zone Infiltration Mass [kg]
+HVAC,Average,Zone Infiltration Mass Flow Rate [kg/s]
+HVAC,Average,Zone Infiltration Air Change Rate [ach]
+
+HVAC,Average,Zone Thermostat Air Temperature [C]
+Zone,Average,Zone Thermostat Control Type []
+HVAC,Average,Zone Thermostat Heating Setpoint Temperature [C]
+HVAC,Average,Zone Thermostat Cooling Setpoint Temperature [C]
+Zone,Average,Zone Adaptive Comfort Operative Temperature Set Point [C]
+HVAC,Average,Zone Predicted Sensible Load Room Air Correction Factor []
+
+HVAC,Sum,Zone Oscillating Temperatures Time [hr]
+HVAC,Sum,Zone Oscillating Temperatures During Occupancy Time [hr]
+HVAC,Sum,Zone Oscillating Temperatures in Deadband Time [hr]
+HVAC,Sum,Facility Any Zone Oscillating Temperatures Time [hr]
+HVAC,Sum,Facility Any Zone Oscillating Temperatures During Occupancy Time [hr]
+HVAC,Sum,Facility Any Zone Oscillating Temperatures in Deadband Time [hr]
+Zone,Sum,Zone Thermal Comfort ASHRAE 55 Simple Model Summer Clothes Not Comfortable Time [hr]
+Zone,Sum,Zone Thermal Comfort ASHRAE 55 Simple Model Winter Clothes Not Comfortable Time [hr]
+Zone,Sum,Zone Thermal Comfort ASHRAE 55 Simple Model Summer or Winter Clothes Not Comfortable Time [hr]
+
+Zone,Sum,Zone Heating Setpoint Not Met Time [hr]
+Zone,Sum,Zone Heating Setpoint Not Met While Occupied Time [hr]
+Zone,Sum,Zone Cooling Setpoint Not Met Time [hr]
+Zone,Sum,Zone Cooling Setpoint Not Met While Occupied Time [hr]
+```
```

Comment:
These Setpoint Not Met Time output variables seem important.  Seems like a clear method for how the space level results get rolled up to the zone level will be needed, eg. average temperature for zone first and then compare to setpoint, or is zone unmet if any spaces are unmet.